### ISSUE TITLE
Expiration email went to admin of the wrong subsite

### ISSUE THREAD

**Issue by tnorthcutt** [restrictcontentpro/restrict-content-pro#1698]
I have a multisite install with several subsites. Today, an (admin) expiration email for a member on site A went to the admin of site B, as if the expired member had been a member of site B.

I haven't been able to reproduce this, and I'm pretty baffled as to how to prevent it in the future... or even how to sort out the root cause.

**Comment by pippinsplugins** [comment]
Were the admins registered on both sites?

**Comment by tnorthcutt** [comment]
@pippinsplugins Nope. The admin that received the email is an admin on site B, but not a user on site A.

However, that same user is an admin on a different subsite C on the network. And, that email address is also the admin notification email for another subsite.

**Comment by mindctrl** [comment]
@tnorthcutt what's your config? Network activated? Using `RCP_NETWORK_SEPARATE_SITES`? Activated on sites individually?

**Comment by tnorthcutt** [comment]
@mindctrl Sorry, I should've included that. We're running it **not** network activated. Activated on sites individually.
